src/Form/NewsletterSubscriptionType.php

```diff
@@ -24,6 +25,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'filter_emojis' => true,
             ])
+            ->add('conditions', CheckboxType::class, [
+                'required' => false,
```

Comment:
Could you please add a `constraints` option like:
```php
->add('conditions', CheckboxType::class, [
    'required' => false,
    'constraints' => new IsTrue(['message' => 'newsletter.conditions.required']),
    'mapped' => false,
])
```
and remove the public property on the object?

I can handle it if you want.